topcoffea/modules/histEFT.py

```diff
@@ -318,20 +318,46 @@ def __reduce__(self):
         )
 
     def make_scalings(self,wc_scalings_lst=None):
+        """
+        This function returns the scalings necessary for running the interference model.
+
+        It obtains all eft coefficients from each bin of the histogram.
+        The coefficients are then normalized to sm, and have the cross terms divided by 2 to obtain lower matrix.
+        (Our histogram saves all elements of the matrix therefore twice of the cross terms)
+
+        lower matrix:
+
+             sm   wc1  wc2  wc3 ...
+        sm    1
+        wc1  c01  c11
+        wc2  c02  c12  c22
+        wc3  c03  c13  c23  c33
+        ...
+
+        scalings for each bin: [1, c01, c11, c02, c12, c22, c03, c13, c23, c33 ... ]
+        If there is a specified list of wcs, this function remaps the coefficients.
```

Comment:
"... this function remaps the coefficients." -> "... this function remaps the coefficients to match the order of the wc_scalings_lst (which may be a subset of all possible WCs)"